### PR TITLE
SAK-44597 Rubrics: Error in console sorting rubrics

### PR DESCRIPTION
**Pull request by MRutea**
SAK-44597 Rubrics: Error in console sorting rubrics

### CODE REVIEW

File: webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubrics-manager.js [hunk 1]
```diff
@@ -143,6 +143,9 @@ class SakaiRubricsManager extends RubricsElement {
 
   sortRubrics(event) {
     const sortInput = event.target.className;
+    if (sortInput === "") {
```
Comment (adrianfish) [inline webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubrics-manager.js:146]: Why not get rid of that toString on sortInput. className is always a string. Also, you can just go if (!sortInput). Empty strings are falsy in JS.
https://developer.mozilla.org/en-US/docs/Web/API/Element/className

File: webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubrics-manager.js [hunk 2]
```diff
@@ -143,6 +143,9 @@ class SakaiRubricsManager extends RubricsElement {
...
+      return;
+    }
     const sortInputValue = sortInput.toString().toLowerCase();
     const rubric = sortInputValue.split('-');
```
Comment (adrianfish) [inline webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubrics-manager.js:150]: I'd just go:
const rubricClass = sortInput && sortInput.toLowerCase().split("-")[0];
You don't even need the falsy test, then.

**Review by adrianfish — CHANGES_REQUESTED** [review]

**Comment by MRutea** [comment]
@adrianfish 
Hi, Adrian;
Thanks for your comments.
I've made the necessary changes.
But finally I think it is convenient to keep the "if" conditional, since I think it adequately prevents possible errors in the successive lines.

Thanks again.

File: webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubrics-manager.js [hunk 3]
```diff
@@ -142,12 +142,16 @@ class SakaiRubricsManager extends RubricsElement {
...
-    const rubricClass = rubric[0];
-    const rubricType = rubric[1];
-    const query = `.${sortInputValue}.sort-element-${rubricClass}`;
+
+    if (!sortInput) {
+      return;
+    }
+
+    const rubricClass = sortInput.toLowerCase().split("-")[0];
+    const rubricType = sortInput.toLowerCase().split("-")[1];
```
Comment (adrianfish) [inline webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubrics-manager.js:153]: const sortInput = event.target.className.toLowerCase();
(see https://developer.mozilla.org/en-US/docs/Web/API/Element/className. className always returns a string)

...

const [rubricClass, rubricType] = sortInput.split("-");
(see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)

const query = `.${sortInput}.sort-element-${rubricClass}`;

**Review by adrianfish — CHANGES_REQUESTED** [review]

**Comment by MRutea** [comment]
@adrianfish 
Hi, Adrian;
Changes made.

I didn't know about "Destructuring assignment" in js.
Thank you again.

**Review by adrianfish — APPROVED** [review]
GTG

**Pull request merged**